bicep/deploy/main.bicep

```diff
@@ -0,0 +1,53 @@
+targetScope = 'subscription'
+
+@allowed([
+  'Dev'
+  'Tst'
```

Comment:
Is  this our standard naming convention for test?

Reply:
Nope - Good catch

Reply:
I'm seeing the trend of using Dev - Staging - Prod more and more... we also use this at a current customer. Should we consider adopting the same?

Reply:
Guess it depends on the customer - for my sake, we can extend the allowed set, but it should always be seen as a starting point, which can be copied and modified to a projects liking.